db/data/controllers/article.controller.js

```diff
@@ -0,0 +1,18 @@
+const allArticlesById = require('../models/articles.models')
+
+const getArticlesById = (request, response) => {
+    const article_id = parseInt(request.params.article_id)
+    console.log(request.params)
+         allArticlesById(article_id).then((article) => {
+            if(!article) {
+                return response.status(404).send({msg: 'Not Found'})
+            }
+        response.status(200).send({article: article})
```

Comment:
Because your key and value are using the same variable, you could short hand this to ```{article}``` and it would work exactly the same.